apps/exit-node/src/exit.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Responsible for creating external requests.
+ */
+import fetch from "node-fetch";
+import { utils } from "rpch-commons/src/index.js";
```

Comment:
thats not how you import the package, do `import { utils } from "rpch-commons";` instead